calico_test/tests/st/test_base.py

```diff
@@ -38,9 +41,17 @@ def setUp(self):
 
         # Delete /calico if it exists. This ensures each test has an empty data
         # store at start of day.
-        subprocess.check_output(
-            "curl -sL http://%s:2379/v2/keys/calico?recursive=true -XDELETE"
-            % self.ip, shell=True)
+        etcd_ssl_type = os.getenv("ETCD_SECURITY_TYPE", None)
+        if etcd_ssl_type == "ssl":
```

Comment:
SSL and TLS are used somewhat interchangeable nowadays.  SSL is the more common name, and usually describes either.

We can't base it off of having the certs dir because the certs dir is not cleaned up after an st-ssl or after run-etcd-ssl.